sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -117,8 +109,7 @@ export class ServiceBusClient {
   createReceiver(
```

Comment:
Remove `options` from the doc string a few lines above

Reply:
Same for all the overloads of `createReceiver`, `createSender` &`createDeadLetterReceiver`